### DISCUSSION TITLE
Is the project abandoned ?

### DISCUSSION THREAD

**Discussion by ghost** [noah-nuebling/mac-mouse-fix#710] [Questions]
Last release and commit were from year ago with no feedback. Is it abandoned ?

**Comment by machinaeZER0** [comment]
The master branch hasn't been updated, but if you select "version 3" in the branch dropdown it looks like there's actually been quite a lot of activity in the background! Excited to see what's been cooking in there :)

**Reply by ghost** [reply]
Yes, it seems something is happening, but there haven't been releases for over a year.
Is @noah-nuebling  actually planning on making a release out of it ?
~~Especially since there are so many unfixed issues in v3 beta?~~
<hr>
Nvm, it seems v2 works better than 3 beta. Also trackpad gestures work just like on the track pad.


So fun. Can't stop switching desktops (left and right) for no reason 😄 . Thank you for making this app @noah-nuebling
